dapp_runner/descriptor/dapp.py

```diff
@@ -49,7 +51,9 @@ class ServiceDescriptor(BaseDescriptor["ServiceDescriptor"]):
     payload: str
     entrypoint: List[List[str]]
     network: Optional[str] = None
+    ip: Optional[List[str]] = field(default_factory=list)  # type: ignore [assignment]  # noqa
```

Comment:
Just a thought: if this is a list, probably `ips` would be better. But maybe not, your decision.

Also I've the same question about `Optional` as in the previous comment, about `depends_on`.